Libplanet.Explorer/Queries/ExplorerQuery.cs

```diff
@@ -25,6 +25,7 @@ public ExplorerQuery(IBlockChainContext<T> chainContext)
             Field<BlockQuery<T>>("blockQuery", resolve: context => new { });
             Field<TransactionQuery<T>>("transactionQuery", resolve: context => new { });
             Field<StateQuery<T>>("stateQuery", resolve: context => new { });
+            Field<VoteQuery<T>>("previousVoteQuery", resolve: context => new { });
```

Comment:
```suggestion
            Field<VoteQuery<T>>("voteQuery", resolve: context => new { });
```

As the type suggests, `voteQuery` seems fine. (or `blockVoteQuery`?)